packages/fxa-payments-server/src/lib/apiClient.ts

```diff
@@ -257,3 +257,52 @@ export async function apiUpdatePayment(params: {
     throw error;
   }
 }
+
+export async function apiCreateCustomer(params: {
```

Comment:
Adding new API client methods for new auth-server routes, while trying to roll in better type annotations.